File: menuoverflow/app/src/main/java/com/example/menuoverflow/MainActivity.java
package com.example.menuoverflow;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public boolean onCreateOptionsMenu(Menu menu){
        getMenuInflater().inflate(R.menu.menuoverflow,menu);
        return true;
    }
    public boolean onOptionsItemSelected(MenuItem item){
        int id=item.getItemId();
        if (id==R.id.op1){
            Toast.makeText(this,"opcion 1", Toast.LENGTH_LONG).show();
        }else if(id==R.id.op2) {
            Toast.makeText(this, "opcion 2", Toast.LENGTH_LONG).show();
        }else if(id==R.id.op3){
            Toast.makeText(this, "opcion 3", Toast.LENGTH_LONG).show();
        }else if(id==R.id.op4){
            Toast.makeText(this, "opcion 4", Toast.LENGTH_LONG).show();
        }
        return super.onOptionsItemSelected(item);

    }
}

File: kotlin/app/src/main/java/com/example/kotlin/MainActivity.kt
package com.example.kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var edad : Int
        var nota : Float = 6.6f
        var nota2 : Double = 5.4
        var letra : Char = 'f'
        var nombre : String = "Madeus"
        var check : Boolean = false
        edad = 35
        var edad2 : Int = 35
        Toast.makeText(this, "la edad es "+edad , Toast.LENGTH_LONG).show()
        val edad3 : Int = 36
        calcular
    }
    fun funcionPrueba(v : View){
        Toast.makeText(this, "hola " , Toast.LENGTH_LONG).show()

    }
}